lib/widgets/autocomplete.dart

```diff
@@ -314,7 +314,11 @@ class _MentionAutocompleteItem extends StatelessWidget {
           mainAxisSize: MainAxisSize.min,
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
-            labelWidget,
+            Row(children: [
+              Flexible(child: labelWidget),
+              if (option case UserMentionAutocompleteResult(:var userId))
+                UserStatusEmoji(userId: userId, size: 18,
+                  padding: const EdgeInsetsDirectional.only(start: 5.0))]),
             if (sublabelWidget != null) sublabelWidget,
           ])),
```

Comment:
I think this new logic is best included as part of `labelWidget`. Conceptually it's closely tied to the user's name, which is part of the label.

Probably in fact the cleanest home for this is in the `switch (option)` above, which is already the place where we inspect the details of `option`.